app/assets/stylesheets/partials/_event_box.scss

```diff
@@ -53,25 +53,23 @@
     }
 
     h4 {
-      margin: 0;
-      font-weight: $font_weight-normal;
-
       @include media(small) { font-size: $font_size-small; }
+      font-weight: $font_weight-normal;
+      margin: 0;
     }
   }
 
   &__event-info {
     @include media(small) { font-size: $font_size-xsmall; }
 
-    span:last-of-type {
-      $spacing: 3px;
-
-      display: inline-block;
-
-      padding-left: $spacing * 2;
-      margin-left: $spacing;
-
-      border-left: 1px solid $color_font-light;
+    span {
```

Comment:
This is also a chance to use a class over an element tag (http://getbem.com/naming/).

Reply:
I added a `event-box__event-info-city` class to the particular span being targeted so that we could follow BEM practice *and* nix a nested pseudo-class.